.github/workflows/build.yml

```
@@ -172,41 +172,41 @@ jobs:
       uses: actions/cache@v3
       with:
         path: /tmp/image
-        key: image-stellar-soroban-rpc-${{ inputs.arch }}-${{ env.SOROBAN_RPC_REPO_BRANCH }}
+        key: image-stellar-stellar-rpc-${{ inputs.arch }}-${{ env.STELLAR_RPC_REPO_BRANCH }}
     - name: Upload Stellar-Core Image
       if: steps.cache.outputs.cache-hit == 'true'
       uses: actions/upload-artifact@v4
       with:
-        name: image-stellar-soroban-rpc-${{ inputs.tag }}-${{ inputs.arch }}
+        name: image-stellar-stellar-rpc-${{ inputs.tag }}-${{ inputs.arch }}
         path: /tmp/image
 
-  build-stellar-soroban-rpc:
-    needs: [load-stellar-soroban-rpc-from-cache]
-    if: ${{ needs.load-stellar-soroban-rpc-from-cache.outputs.cache-hit != 'true' }}
+  build-stellar-stellar-rpc:
+    needs: [load-stellar-stellar-rpc-from-cache]
+    if: ${{ needs.load-stellar-stellar-rpc-from-cache.outputs.cache-hit != 'true' }}
     runs-on: ${{ inputs.arch == 'arm64' && 'ubuntu-jammy-4-cores-arm64' || 'ubuntu-latest' }}
     steps:
     - id: cache
       uses: actions/cache@v3
       with:
         path: /tmp/image
-        key: image-stellar-soroban-rpc-${{ inputs.arch }}-${{ env.SOROBAN_RPC_REPO_BRANCH }}
+        key: image-stellar-stellar-rpc-${{ inputs.arch }}-${{ env.STELLAR_RPC_REPO_BRANCH }}
     - if: inputs.arch == 'arm64'
       uses: docker/setup-qemu-action@e81a89b1732b9c48d79cd809d8d81d79c4647a18
       with:
         platforms: arm64
     - uses: docker/setup-buildx-action@5146db6c4d81fbfd508899f851bbb3883a96ff9f
-    - name: Build Stellar-Soroban-Rpc Image
+    - name: Build Stellar-stellar-rpc Image
```

Won't have a functional impact, but in a few places this needs a little tidying:
```suggestion
    - name: Build Stellar-Rpc Image
```